### PR TITLE
Refactor about the output

### PR DESCRIPTION
**Pull request by juanmanuelramallo**
Introduced a new way for task objects to print information by using the
Output object and its struct.

### CODE REVIEW

File: lib/rsgem/version.rb [hunk 1]
```diff
@@ -3,6 +3,6 @@
 module RSGem
   MAJOR = 0
   MINOR = 4
-  PATCH = 0
+  PATCH = 1
```
Comment (santib) [inline lib/rsgem/version.rb]: dropping support to older ruby versions is a patch change? 🤔

File: lib/rsgem/tasks/output.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RSGem
+  module Tasks
+    #
+    # Structure to bundle all messages from a task
+    #
+    OutputStruct = Struct.new(:name, :success, :warning, :error, keyword_init: true)
```
Comment (santib) [inline lib/rsgem/tasks/output.rb:8]: I think I'd call it `Output` not `OutputStruct`. That's like calling a string `OutputString` and I don't think that's a common practice in the ruby world at least 👀

**Review by santib — APPROVED** [review]

File: lib/rsgem/version.rb (hunk 1 above)
Comment (juanmanuelramallo) [reply lib/rsgem/version.rb]: lol good catch, this change shouln't be here. 

I usually merge changes and after that I prepare a version change commit

**Review by martinjaimem — APPROVED** [review]

**Review by JuannFerrari — APPROVED** [review]
LGTM, nice cleaning 🧹

**Pull request merged**